scripts/smoke_test_examples.py

```
@@ -0,0 +1,388 @@
+Smoke test script for FinTorch examples.
+
+This script runs example files to check if they can successfully reach the training stage
+without actually running the full training process. It monitors the output for training-related
+keywords and terminates the process once training is detected or after a timeout.
+"""
+
+import os
+import subprocess
+import sys
+import time
+import signal
```

Ruff (0.11.9)

`signal` imported but unused

Remove unused import: `signal`

(F401)